scripts/build_industrial_production_per_country.py

```diff
@@ -179,8 +179,8 @@ def industry_production(countries):
     return demand
 
 
-def add_ammonia_demand_separately(demand):
-    """Include ammonia demand separately and remove ammonia from basic chemicals."""
+def separate_basic_chemicals(demand):
+    """Remove ammonia, chlorine and methanol from basic chemicals to get HVC."""
```

Comment:
```suggestion
    """Separate basic chemicals into ammonia, chlorine, methanol and HVC."""
```
HVC is not the rest in the current implementation, but has its own input in the `config.yaml`.